cli/index.js

```diff
@@ -1,13 +1,16 @@
 var program = require('commander'),
     chalk = require('chalk'),
     wake = require('../'),
+    relay = require('../').relay,
     isMACValid = require('../').isMACValid;
 
 program
     .usage('<mac>')
     .option('--ip [ip]', 'IP Address [255.255.255.255] of target computer', '255.255.255.255')
     .option('--host [host]', 'Hostname of the target computer')
     .option('--port [port]', 'Port [9]', 9)
+    .option('--relay', 'Listen for packet and then broadcast it')
+    .option('--relayport [port]', 'Listen for relay packet on this port')
```

Comment:
You should validate user input here. For example, it should be set with value (not boolean), maybe casted to number. I didn't write whose checks initially, but i think this should be done — https://github.com/tj/commander.js#coercion :)

Reply:
I looked at the documentation, and found this example which is similar:

...
  .option('-v, --verbose', 'A value that can be increased', increaseVerbosity, 0)
...

Is this what you had in mind?  I can add that to the code.

Reply:
Specifically, for the --relay option